core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -183,8 +193,14 @@ private synchronized void computeSchedulable(List<Flow> flows) {
 
         this.schedulable = flows
             .stream()
-            .filter(flow -> flow.getTriggers() != null && flow.getTriggers().size() > 0)
+            .filter(flow -> flow.getTriggers() != null && !flow.getTriggers().isEmpty())
             .filter(flow -> !flow.isDisabled())
+            .peek(flow -> {
+                if (flow instanceof FlowWithException fwe) {
```

Comment:
I don't think it's necessary, the filtering must be done on the FlowListeners (already done on the JDBC but not on the Kafka as I see)